### DISCUSSION TITLE
U1 use with phone

### DISCUSSION THREAD

**Discussion by cjkoral** [NightscoutFoundation/xDrip#3505] [General]
Hello, I am in the U1 beta program, and am looking to have my watch read from the phone (or vise versa) and then if my phone is out of range the watch still connects, (GTS does this). Do you have any advice on how to achieve this with the U1? 

Thank you!

**Comment by Navid200** [comment]
U1 and xDrip on the phone do not communicate with each other directly .   At least, that's how it is now.

What is your limitation?
What is really the question?
Can you explain what you want to do that isn't working?

**Reply by cjkoral** [reply]
@Navid200 thank you for the response.

Behavior I am getting:  Either the watch, or the phone will give me my glucose readings.  Sometimes Ill look at my watch, and it will not be connected, so i have to find my phone in the house to check my readings.  It seems that the watch and phone both compete for the same slot on the receiver.

Behavior I hope to get: Both the watch and the phone receive the readings, I can look at my watch, or my phone and they will both be up to date with the latest reading (assuming no error)

**Reply by Navid200** [reply]
You should communicate your findings to the beta test program you are a part of.  

I do what I can to help if I can here.
What I state is always my personal opinion:

GTS could not upload to the Cloud.  Neither could it collect from G7.  U1 can do both.  
Is there a reason you need to use both the phone and the watch simultaneously?

**Reply by Navid200** [reply]
Is the device G6 or G7?

**Reply by cjkoral** [reply]
Hi, it's a g6, the watch battery life is less than a day so though I usually check using the watch, when it dies It takes some time for the phone to start receiving readings

**Reply by Navid200** [reply]
U1 and xDrip on the phone are meant to coexist happily collecting from the same G6 or G7.
You may need to wait up to 20 minutes for the second device (phone or U1) to pair.
After both U1 and phone have paired, there should be no issue and they should not compete.

Please send feedback to the company.
